onnxruntime/contrib_ops/cpu/quantization/matmul_nbits.cc

```diff
@@ -193,10 +193,39 @@ Status MatMulNBits<T1>::PrePack(const Tensor& tensor, int input_idx, /*out*/ All
     auto qptr = tensor.DataRaw();
     auto scale_ptr = scales ? scales->DataRaw() : nullptr;
     packed_b_ = IAllocator::MakeUniquePtr<void>(alloc, packed_b_size_, true);
+#if defined(MLAS_TARGET_ARM64)
+    if (nbits_ == 8) {
+      // On ARM64, the scales should NOT be provided while packing the weights for nbits_ == 8.
+      MlasQNBitGemmPackQuantBData(N_, K_, nbits_, block_size_, compute_type_, qptr, packed_b_.get(), nullptr,
```

Comment:
is it possible for the ARM64 8b implementation to ignore the scales? then maybe we can simplify this call site